win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1148,6 +1149,16 @@ BITMAP_LoadImageW(
         goto end;
     CopyMemory(pbmiCopy, pbmi, iBMISize);
 
+    /* Test if MSVC Resource Compiler used and if so, adjust pvBits */
+    HdrSize = *(char*)pbmi;
+    if (pbmiCopy->bmiHeader.biSizeImage + HdrSize + 12 == ResSize
+             && compr == BI_BITFIELDS && bpp == 32)
```

Comment:
For BI_BITFIELDS , MSDN says "This is valid when used with 16- and 32-bpp bitmaps".

Reply:
Based on testing with a 16-bit BI_BITFIELD, no further code adjustment seem needed. As you can see below in a similar comparison to what I showed above, the MSVC compiled comctl32.dll and the GCC compiled comctl32.dll have the same internal size as shown by Resource Hacker Version 5.2.4.

![BI_BITFIELDS_16_BPP_comctl32_dll_OK](https://github.com/reactos/reactos/assets/32620577/118d378e-f9c1-49cb-9db9-8bd81f68a937)